variables.tf

```diff
@@ -179,3 +179,9 @@ variable "self_service_saml_provider_arn" {
   type        = string
   default     = null
 }
+
+variable "session_timeout_hours" {
+  description = "The maximum session duration is a trigger by which end-users are required to re-authenticate prior to establishing a VPN session. Default value is 24."
```

Comment:
@rpadovani From the [docs](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ec2_client_vpn_endpoint#session_timeout_hours), there are only certain values that are valid. Can you please enumerate those in the description so others don't try to pass whatever number they want and run into a validation error?